Client/effekseer/CEffekseerEffectHandlerImpl.cpp

```diff
@@ -71,3 +76,68 @@ float CEffekseerEffectHandlerImpl::GetDynamicInput(int32_t index)
 {
     return m_pInterface->GetDynamicInput(m_Handle, index);
 }
+
+void CEffekseerEffectHandlerImpl::SetRandomSeed(int32_t seed)
```

Comment:
Is there a reason for creating a whole interface over effekseer?

Reply:
> Is there a reason for creating a whole interface over effekseer?

This is an adapter between MTA types and Effekseer types.
* Main integration part is under one project.
* From an architecture perspective that makes future vendor updates much easier.
* Effekseer namespaces brings an additional mess in MTA projects.
* Dependency on interfaces is better than on implementation.

> I guess it's so than effekseer can be a dll, but I don't think there's any benefit from that.

I use dynamic linking for Effekseer over static because i need that Interface in both Client Core and Client Deathmatch projects. I can be heavily wrong, but that cause compiled code duplication in this projects with static linking. I saw this two years ago, when i was truing to compile Effekseer first time. I didn't remember all details.

I would prefer to link Effekseer with Client core project, when i can. I don't know if this is possible yet.